scripts/release.js

```diff
@@ -168,17 +173,36 @@ process.chdir(path.join(__dirname, '../dependency-check-app'));
 spawnOrFail('npm', ['init -y']);
 spawnOrFail('npm', ['install react react-dom']);
 spawnOrFail('npm', [`install amazon-chime-sdk-js@${updatedSdkVersion} styled-components styled-system`]);
-spawnOrFail('yalc', ['add amazon-chime-sdk-component-library-react']);
-checkWarning('npm', ['install -q'], null, 'amazon-chime-sdk-component-library-react');
+checkWarning('npm', [`install ../amazon-chime-sdk-component-library-react-${versionString}.tgz`], null, 'amazon-chime-sdk-component-library-react');
```

Comment:
Why do we want to remove the `-q` argument ?

Reply:
I am not sure what is the use of `-q` and if it is related to `yalc`. I think installing the tar file is enough, so I removed the `-q`